packages/contracts/src/guise/LibGuiseLevel.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { ActiveGuise, ActiveGuiseTableId } from "../codegen/Tables.sol";
+import { GuisePrototype, GuisePrototypeTableId } from "../codegen/Tables.sol";
+
+import { PStat_length } from "../CustomTypes.sol";
+import { LibExperience } from "../charstat/LibExperience.sol";
+
+library LibGuiseLevel {
+  /// @dev Get target's aggregate level using its guise's level multipliers.
+  /// (aggregate level means all primary stats aggregated together)
+  function getAggregateLevel(bytes32 targetEntity) internal view returns (uint32) {
+    uint256 guiseProtoEntity = ActiveGuise.get(targetEntity);
+    uint32[PStat_length] memory levelMul = GuisePrototype.get(bytes32(guiseProtoEntity));
```

Comment:
the typecast is unnecessary

Reply:
I tried to write without type casting, but with `pnpm run test`, the compiler reported an error of the arguments being passed, since `GuisePrototype.get` accepts `bytes32`, and ActiveGuise.get returns `uint256`
<img width="1038" alt="Снимок экрана 2023-06-05 в 16 11 37" src="https://github.com/dk1a/wanderer-cycle2/assets/107617521/5dba7cbf-947f-4b8c-9b04-306fd4e0d7ba">

Reply:
oh right I changed the tables to `bytes32` in https://github.com/dk1a/wanderer-cycle2/pull/2